redbot/cogs/audio/audio.py

```diff
@@ -1909,6 +1930,14 @@ def _track_creator(player, position=None, other_track=None):
             track_obj[key] = value
         return track_obj
 
+    @staticmethod
+    def _thumbnail(track):
+        if "youtube" in track.uri:
+            values = track._info.values()
+            id = list(values)[1]
```

Comment:
Will this always be the second item in `values()`? Would it have a key instead? Also, is it possible to patch Lavalink for this?

Reply:
Changed it up a bit. And the thumbnail fetching maybe should be moved to Red-Lavalink.